sunpy/map/tests/test_map_factory.py

```diff
@@ -152,6 +153,15 @@ def test_errors(self):
             # Check a random unsupported type (int) fails
             sunpy.map.Map(78)
 
+        # If one file failed to load, make sure it's listed in warning msg
+        nonexist_file = os.path.join(filepath, 'nonexist.fits')
+        files = [AIA_171_IMAGE, nonexist_file]
+        with pytest.warns(SunpyUserWarning, match='Error reading file {0}'.format(nonexist_file)):
+            try:
+                sunpy.map.Map(files)
+            except Exception:
+                pass
```

Comment:
Why was this put into a try block?

Reply:
Second file doesn't exist. So it will throw a value error. Feature added was to provide warning before error. To validate what happened before error , I put into a try block.

Reply:
The value error should be caught by pytest and not by the try block. Please see https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions

Reply:
Okay